ci/build_cuda_parallel_python.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -euo pipefail
+
+# Get the Python version from the command line arguments -py-version=3.10
+py_version=${2#*=}
+echo "Docker socket: " $(ls /var/run/docker.sock)
+
+# given the py_version build the wheel and output the artifact
+# to the artifacts directory
+docker run --rm -i \
```

Comment:
Need some comments to explain why/what is happening in this `docker run` command.